data/sgid-transportation/roads-system/index.html

```diff
@@ -124,6 +124,7 @@ <h5 id="RoadCenterlines-updates">Updates</h5>
 <p>Updates to this dataset are published every two months with approximately 5 counties' roads receiving detailed review during each update cycle.</p>
 <p>Recent update summaries:</p>
 <ul>
+<li><a href="{{site.baseurl}}{% post_url 2016-10-03-utah-sgid-statewide-roads-data-layer-updates %}">October 03, 2016</a></li>  
```

Comment:
Please try to be consistent with your post names. Pick a format and stick with it.